# International investment position
### market value by currency, quarterly



This presentation contains the description of the data base from [Government of Canada](https://open.canada.ca/data/en/dataset/67e5b0f7-6307-4e13-8432-c4cd1e2e1adc).

Canada’s international investment position (IIP) measures the value of:

* Foreign assets owned by its residents (external assets)
* Domestic assets owned by foreigners (external liabilities)
* Net investment position is the difference between what Canada owns (its external assets) and what Canada owes (its external liabilities) to non-residents.


In [10]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

In [12]:
# add data
df = pd.read_csv('./36100412.csv')
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 3014 entries, 0 to 3013
Data columns (total 17 columns):
 #   Column                                      Non-Null Count  Dtype  
---  ------                                      --------------  -----  
 0   REF_DATE                                    3014 non-null   object 
 1   GEO                                         3014 non-null   object 
 2   DGUID                                       3014 non-null   object 
 3   Canada's international investment position  3014 non-null   object 
 4   Geographic region                           3014 non-null   object 
 5   Currency                                    3014 non-null   object 
 6   UOM                                         3014 non-null   object 
 7   UOM_ID                                      3014 non-null   int64  
 8   SCALAR_FACTOR                               3014 non-null   object 
 9   SCALAR_ID                                   3014 non-null   int64  
 10  VECTOR      

As we can see, this data base contains 17 variables and 3014 observations, from those variables the ones that are must of our interes are:

* REF_DATE, which contains the quarter of the year
